etc/torbrowser-launcher.profile

```diff
@@ -48,7 +48,7 @@ shell none
 #tracelog
 
 disable-mnt
-private-bin bash,cat,cp,cut,dirname,env,expr,file,getconf,gpg,grep,gxmessage,id,kdialog,ln,mkdir,mv,pwd,python*,readlink,realpath,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
+private-bin bash,cat,cp,cut,dirname,env,expr,file,gpg,grep,gxmessage,kdialog,ln,mkdir,mv,python*,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
```

Comment:
Some of those were added by @pizzadude in initial profile: https://github.com/netblue30/firejail/commit/255b087f869bd8ecb8135c3452a84b262fdfb7ef#diff-0443272f6d05e5603c367ad95b8676ea

Some were added by @SkewedZeppelin : https://github.com/netblue30/firejail/commit/78bb84ddf277dab653a08f97303894e35433402f#diff-0443272f6d05e5603c367ad95b8676eaR35

@SkewedZeppelin does your approval means they are no longer needed?